### ISSUE TITLE
The app crashed after delete my account

### ISSUE THREAD

**Issue by akzhar** [getinsomnia/insomnia#6589]
### Expected Behavior

Sign-up / Sign-in window should appears to continue using the app

### Actual Behavior

Right after start an app the following error pops-up:
```
Application Error
Failed to render. Please report to [our Github Issues](https://github.com/Kong/insomnia/issues)

Network connectivity issue: Failed to load Organizations. Cannot read properties of undefined (reading 'find')
Try to reload the app
Error: Network connectivity issue: Failed to load Organizations.
Cannot read properties of undefined (reading 'find') at Module.Il1 [as indexLoader]
(file:///C:/Users/Katana/AppData/Local/insomnia/app-8.0.0/resources/app.asar/index-334620de.js:1255:3095) at async d8
(file:///C:/Users/Katana/AppData/Local/insomnia/app-8.0.0/resources/app.asar/index-334620de.js:31:3691) at async Promise.all (index 2) at async ce
(file:///C:/Users/Katana/AppData/Local/insomnia/app-8.0.0/resources/app.asar/index-334620de.js:30:21935) at async qe
(file:///C:/Users/Katana/AppData/Local/insomnia/app-8.0.0/resources/app.asar/index-334620de.js:30:16866) at async we
(file:///C:/Users/Katana/AppData/Local/insomnia/app-8.0.0/resources/app.asar/index-334620de.js:30:15042)
```

### Reproduction Steps

1. Download last 8.0 installer `Insomnia.Core-8.0.0.exe`
2. Install / run the app
3. Successfully sign-up / sign-in using Google account (set passphrase etc.)
4. Go to [account settings page](https://app.insomnia.rest/app/settings/account)
5. Hit `Delete my account` button and confirm
6. Return to the app - an above error appears

From this point the app crashed and doesen't work at all.
There is no way to log-in using another account or somehow force the app to work.
The following steps was sttempted to try solve the problem - nothing helped me:
- Hitting `Try to reload the app` button below the error message
- End task through Windows Task Manager and run the app again
- Uninstall the app / install again 8.0

I am only managed to solve this by downgrading to another version, in my case [Insomnia 2022.6.0](https://github.com/Kong/insomnia/releases/tag/core%402022.6.0)

### Is there an existing issue for this?

- [X] I have searched the [issue tracker](https://www.github.com/Kong/insomnia/issues) for this problem.

### Insomnia Version

8.0.0

### What operating system are you using?

Windows

### Operating System Version

Windows 11 Pro Version 22H2 OS build 22621.2283

### Installation method

Download from insomnia.rest

**Comment by subnetmarco** [comment]
This issue will be fixed in 8.1 today or tomorrow the latest.

**Comment by subnetmarco** [comment]
Fixed as 8.1 is now out.
